### ISSUE TITLE
Invitations - Remove space bar as confirmation

### ISSUE THREAD

**Issue by Jackseed** [blockframes/blockframes#5217]
url: https://staging.archipelmarket.com/c/o/dashboard/event/h91BbmBH9OMDnazNLwi3/edit
user dev+brandyn-xio@blockframes.io

When writing emails or name to invite people, the space bar is considered as confirmation. It's really annoying as you can search someone by his/her name so you want to be able to write both first name, space bar, last name. 
This space bar behaviour should simply be removed.
![image.png](https://images.zenhubusercontent.com/5c6a89ac8de1b31d5443ab99/579f46e6-ceff-4073-bf15-be0460f291ca)

**Comment by RemcoSimonides** [comment]
Side effect of my fix was that 
* copy pasting multiple email addresses with SPACE as separator doesn't work anymore.
This change has now been reverted on production to not block copy/pasting multiple email addresses with SPACE as separator.

An idea Vincent was to distinguish between copy/paste event on the input field and typing. In case of typing, the SPACE isn't considered as a separator and in the case of copy/paste it is considered as a separator.

I reopened this issue so we can look into how we can solve this in a better way @Jackseed
